stable/postgresql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: postgresql
-version: 0.17.0
+version: 0.17.1
```

Comment:
since this is new functionality please bump to `0.18.0` thanks!

Reply:
Sure, bumped it in latest commit.